tests/unit/statistical_testing/KernelSelection_unittest.cc

```diff
@@ -287,7 +287,7 @@ TEST(KernelSelectionMaxCrossValidation, quadratic_time_single_kernel_dense)
 	mmd->set_train_test_mode(false);
 
 	auto selected_kernel=static_cast<CGaussianKernel*>(mmd->get_kernel());
-	EXPECT_NEAR(selected_kernel->get_width(), 0.0625, 1E-10);
+	EXPECT_NEAR(selected_kernel->get_width(), 0.03125, 1E-10);
```

Comment:
@karlnapf @lambday just need your ok here... i think it's fine the diff is just due to the fact that the prng behaviour changed, hence even setting the same seed will not give you back the same series of randoms....